website/docs/r/cluster.html.markdown

```diff
@@ -51,3 +51,14 @@ In addition to arguments listed above, the following attributes are exported aft
 * `id` - Cluster identifier.
 
 * `cluster_info` - Information about cluster like name, state, host instance type and cluster identifier.
+
+## Import
```

Comment:
Should we add a note that the tf file should have the corresponding empty resource for this to be successful?

Reply:
@rchadalawada good point. But I think that is a given when you are familiar with the terraform commands. If I don't add an empty resource block I see the following error from terraform  

resource address "vmc_srm_node.srm_node_1" does not exist in the configuration.

Before importing this resource, please create its configuration in the root module. For example:

resource "vmc_srm_node" "srm_node_1" {
  # (resource arguments)
}